kubernetes/product-deployment/scripts/cleanup.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Shall we add proper License headers to all the script files?

Reply:
Added.